static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -0,0 +1,105 @@
+import styled from '@emotion/styled';
+
+import {AreaChart, AreaChartProps} from 'sentry/components/charts/areaChart';
+import {MemorySpanType} from 'sentry/components/events/interfaces/spans/types';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {formatBytesBase2} from 'sentry/utils';
+import {getFormattedDate} from 'sentry/utils/dates';
+import theme from 'sentry/utils/theme';
+import EmptyMessage from 'sentry/views/settings/components/emptyMessage';
+
+type Props = {
+  memorySpans: MemorySpanType[] | undefined;
+};
+
+const formatTimestamp = timestamp =>
+  getFormattedDate(timestamp * 1000, 'MMM D, YYYY HH:mm:ss');
+
+function MemoryChart({memorySpans = []}: Props) {
+  if (memorySpans.length <= 0) {
+    return <EmptyMessage>{t('No memory metrics exist for replay.')}</EmptyMessage>;
+  }
+
+  const chartOptions: Omit<AreaChartProps, 'series'> = {
+    grid: {
+      left: '10px',
+      right: '10px',
+      top: '40px',
+      bottom: '10px',
+    },
+    tooltip: {
+      trigger: 'axis',
+      valueFormatter: (value: number | null) => formatBytesBase2(value || 0),
+    },
```

Comment:
Ah I forgot to mention this earlier but we have a bunch chart components that setup defaults for all of these chartOptions. Could we try using them when possible? they're in `app/components/charts/components`. (let's do this as a follow-up PR)

Reply:
Will post a ticket in this comment when I make it.